docs/using-lagoon-advanced/backups.md

```diff
@@ -25,3 +25,8 @@ Backups stored in Restic will be tracked within Lagoon, and can be recovered via
 
 Backups of development environments are attempted nightly and are strictly a best effort service.
 
+## Custom Backup and/or Restore Locations
+
+Lagoon supports custom backup and restore locations via the use of the "[Custom Backup Settings](https://github.com/uselagoon/lagoon/blob/main/docs/using-lagoon-advanced/environment-variables.md#custom-backup-settings)" and/or "[Custom Restore Settings](https://github.com/uselagoon/lagoon/blob/main/docs/using-lagoon-advanced/environment-variables.md#custom-restore-settings)" variables stored in the Lagoon API for each project.
+
+***Note that using these variables may affect any SLAs in effect for Lagoon managed backups***
```

Comment:
Does Lagoon the open source project have SLAs? Not sure where we are with Lagoon vs amazee.io docs, but maybe this note makes more sense in amazee.io? @tobybellwood or @AlannaBurke?

Reply:
This concern was exactly why I preferred to leave this more vague. The fact of the matter is that once any user of a Lagoon project chooses to own the backup and/or restore locations, we can no longer rely on Lagoon automatically creating and maintaining access to them. If you've a better to way to say that, please feel free to suggest an edit. :)

Reply:
How about:
***Please note that these settings are for use with open-source Lagoon installations only. If you are an amazee.io customer, please see your amazee.io administrator before making these changes.***

Reply:
@cdchris12 Since it looks like you'll have to resolve a conflict anyway... we settled on the following language;

> Proceed with caution: Setting these variables will override backup/restore storage locations that may be configured at a cluster level. Any misconfiguration will cause backup/restore failures.

Would that still cover your initial concerns?

Reply:
That works for me. I'll commit that now :)